views/partials/navbar.ejs

```diff
@@ -3,12 +3,23 @@
 
   <div class="nav-links">
     <% if(!currentUser) {%>
-    <a href="/login">Login</a>
-    <a href="/register">Register</a>
+    <a class="nav-link-guest" href="/login">Login</a>
+    <a class="nav-link-guest" href="/register">Register</a>
     <% } else { %>
-    <a href="/play">Play</a>
-    <a href="/account">My Account</a>
-    <a href="/logout">Log out</a>
+    <a class="nav-link" href="/play">Play</a>
+    <a class="nav-link" href="/account">My Account</a>
+    <a class="nav-link" href="/logout">Log out</a>
+
+    <a class="icon-wrapper" href="/play"
```

Comment:
Why not have these icons elements inside `a` elements above?

Reply:
Thats a good point, didnt realize this while doing it. Will change it, thanks!